frontend/www/js/omegaup/problem/mine.ts

```diff
@@ -83,5 +84,52 @@ OmegaUp.on('ready', () => {
       .catch(UI.apiError);
   }
 
+  function visibilityTrue(
+    newVisibility: number,
+    oldVisibility: number,
+  ): number {
+    let visibility: number = 0;
+    if (newVisibility == 1) {
+      switch (oldVisibility) {
+        case payload.visibilityStatuses['privateBanned']: {
+          visibility = payload.visibilityStatuses['publicBanned'];
+          break;
+        }
```

Comment:
```suggestion
        case payload.visibilityStatuses['privateBanned']:
          return payload.visibilityStatuses['publicBanned'];
```

igual en todos los demás casos. eso quiere decir que L91 no es necesaria!

Reply:
listo